tests/Jellyfin.Networking.Tests/NetworkParseTests.cs

```diff
@@ -53,31 +94,74 @@ public void IgnoreVirtualInterfaces(string interfaces, string lan, string value)
             using var nm = new NetworkManager(GetMockConfig(conf), new NullLogger<NetworkManager>());
             NetworkManager.MockNetworkSettings = string.Empty;
 
-            Assert.Equal(nm.GetInternalBindAddresses().AsString(), value);
+            Assert.Equal(value, nm.GetInternalBindAddresses().AsString());
         }
 
         /// <summary>
-        /// Checks IP address formats.
+        /// Tests IP addresses are NOT part of, or have been excluded from the network range provided.
+        /// </summary>
+        /// <param name="network">Network ranges.</param>
+        /// <param name="value">Value to check.</param>
+        [Theory]
+        [InlineData("192.168.10.0/24,!192.168.10.60/32", "192.168.10.60")]
+        [InlineData("www.google.com", "")]
+        public void Is_Not_In_Network(string network, string value)
```

Comment:
This is already tested in `NetworkManagerTests.cs`

Reply:
removed